src/services/uploadFile.js

```diff
@@ -2,7 +2,7 @@ import { ref, uploadBytesResumable, getDownloadURL } from 'firebase/storage';
 
```

Comment:
@Lelian248 @omarbanat to keep you updated, I changed the file and function name from image to file, since I used it for audio files also, so to not have duplicate code